### ISSUE TITLE
Fix "breadcrumb" trail for consistency

### ISSUE THREAD

**Issue by pombase-admin** [pombase/canto#824]
The trail of links is currently inconsistent and not as useful as it should be.

eg. on the publication page it shows:

-> PomBase Canto
 -> PMID:7462672 summary

it you click on a gene name it adds that gene to the list:
   -> cdc1

but it you click on the Genotype management link on the publication page it doesn't add "Genotype management" to the list.

There are other cases too.

I'm putting this at priority 6 so it doesn't get completely forgotten.

Original comment by: kimrutherford

**Comment by ValWood** [comment]
From Midori:
It's just occurred to me that maybe what Val & Antonia are asking for is for the breadcrumb trail to show the actual path that you *did* follow, whether it is or isn't the only way to get from here to there.
.... Unless there's any worry that "path I followed" would be mistaken for "only path there is"? 

Val
I presumed the  breadcrumb trail showed me where I came from....

**Comment by kimrutherford** [comment]
Currently it's definitely a poor mans breadcrumb trail.  It doesn't show you where you came from consistently.  It works better before the multi-allele changes because there were less paths through the webapp so the trail could be easily faked.

We'll have to take a bit of care because if the trail faithfully shows where you've been it could get arbitrarily long. ("summary"->"gene"->"summary"->"genotype manager"->"genotype"->"summary"->...")

**Comment by kimrutherford** [comment]
From #908: the end of the breadcrumb trail on the genotype and gene pages shouldn't be a link.

**Comment by ValWood** [comment]
related to https://github.com/pombase/canto/issues/884

**Comment by ValWood** [comment]
Kim will make this consistent

**Comment by kimrutherford** [comment]
> Kim will make this consistent

First fix: I've added genotype management to the trail.

**Comment by ValWood** [comment]
Is there anything still to do here?

**Comment by kimrutherford** [comment]
Yep, there are still bugs.  For example if you click on a curation type from a gene, the trail doesn't change. But if you type in the autocomplete and then choose a term, two items get added to the trail.

**Comment by ValWood** [comment]
Shall we close this?
